### PR TITLE
[ExoBundle] bugfixes

### PR DESCRIPTION
**Pull request by stefk**
Trying to solve #568 build issue.

### CODE REVIEW

File: plugin/exo/Resources/public/js/angular/Image/Services/ImageAreaService.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ ImageAreaService.prototype.isInArea = function isInArea(areaDefinition, coords)
 
             break;
         case 'circle':
-            if (Math.sqrt(coords.x - area.center.x) + Math.sqrt(coords.y - area.center.y) <= Math.sqrt(area.radius)) {
+            if (Math.sqrt((coords.x - area.center.x)*(coords.x - area.center.x) + (coords.y - area.center.y)*(coords.y - area.center.y)) <= area.radius) {
```
Comment (stefk) [inline plugin/exo/Resources/public/js/angular/Image/Services/ImageAreaService.js]: Missing spaces around `*` operand

File: plugin/exo/Resources/public/js/angular/Question/Controllers/Type/ChoiceQuestionCtrl.js [hunk 2]
```diff
@@ -8,9 +8,12 @@ var ChoiceQuestionCtrl = function ChoiceQuestionCtrl(FeedbackService, ChoiceQues
     AbstractQuestionCtrl.apply(this, arguments);
     
     this.ChoiceQuestionService = ChoiceQuestionService;
-
+
```
Comment (stefk) [inline plugin/exo/Resources/public/js/angular/Question/Controllers/Type/ChoiceQuestionCtrl.js]: Trailing whitespace

File: plugin/exo/Resources/public/js/angular/Question/Controllers/Type/ChoiceQuestionCtrl.js [hunk 3]
```diff
@@ -8,9 +8,12 @@ var ChoiceQuestionCtrl = function ChoiceQuestionCtrl(FeedbackService, ChoiceQues
     AbstractQuestionCtrl.apply(this, arguments);
     
     this.ChoiceQuestionService = ChoiceQuestionService;
-
+    
     if (this.question.choices) {
         this.choices = this.question.choices;
+        for (var i = 0; i < this.choices.length; i++) {
+            this.choices[i].valid = 0;
```
Comment (stefk) [inline plugin/exo/Resources/public/js/angular/Question/Controllers/Type/ChoiceQuestionCtrl.js]: Why is the value of `valid` an integer here? It seems it should be a boolean.

File: plugin/exo/Resources/public/js/angular/Question/Controllers/Type/ClozeQuestionCtrl.js [hunk 4]
```diff
@@ -8,6 +8,10 @@ var ClozeQuestionCtrl = function ClozeQuestionCtrl(FeedbackService, ClozeQuestio
     AbstractQuestionCtrl.apply(this, arguments);
 
     this.ClozeQuestionService = ClozeQuestionService;
+    
+    for (var i = 0; i < this.question.holes.length; i++) {
+        this.question.holes[i].valid = 0;
```
Comment (stefk) [inline plugin/exo/Resources/public/js/angular/Question/Controllers/Type/ClozeQuestionCtrl.js]: Same (int vs bool)

File: plugin/exo/Resources/public/js/angular/Question/Services/Type/ChoiceQuestionService.js [hunk 5]
```diff
@@ -112,11 +116,12 @@ ChoiceQuestionService.prototype.isChoiceValid = function isChoiceValid(question,
...
 
     var choiceSolution = this.getChoiceSolution(question, choice);
-    if (choiceSolution.score > 0) {
+    console.log(choiceSolution);
+    if (choiceSolution.rightResponse) {
         // The current choice is part of the right response => User choice is Valid
         isValid = true;
     }
-
+console.log(isValid);
```
Comment (stefk) [inline plugin/exo/Resources/public/js/angular/Question/Services/Type/ChoiceQuestionService.js]: Should be removed

File: plugin/exo/Resources/public/js/angular/Question/Controllers/Type/ChoiceQuestionCtrl.js (hunk 3 above)
Comment (Elorfin) [inline plugin/exo/Resources/public/js/angular/Question/Controllers/Type/ChoiceQuestionCtrl.js]: I need to check, but if I remember correctly there are 3 values possible : 0, 1, 2. Where 1 or 2 is when there are missing choices in multiple.

So the boolean is not sufficient in this case. Using some class constants can help for readability.

Comment (stefk) [inline plugin/exo/Resources/public/js/angular/Question/Controllers/Type/ChoiceQuestionCtrl.js]: +1 for constants or es5 equivalent

**Comment by Elorfin** [comment]
LGTM

**Comment by stefk** [comment]
LGTM

**Pull request merged**